plugins/polio/js/src/domains/Calendar/campaignCalendar/Filters.js

```diff
@@ -63,6 +63,28 @@ const Filters = ({
         dateApiToDateRangePicker(params.roundStartTo),
     );
 
+    const filtersFilled = useCallback(() => {
```

Comment:
cool, no i have the sentiment that you can remove totally the usecallback, the value returned is a boolean and don't even need to be memofied, just make it a bool